instagram-app/src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,44 +1,47 @@
 import React, { Component } from 'react'
-import { FaSearch, FaRegCompass, FaRegHeart, FaRegUser, FaInstagram} from 'react-icons/fa'
+import { FaRegCompass, FaRegHeart, FaRegUser, FaInstagram} from 'react-icons/fa'
 import {IconContext} from 'react-icons'
 import PropTypes from 'prop-types'
 import './SearchBar.scss'
+import {ButtonMain as Button} from '../../styled/Buttons'
+import {Logo, Header, SearchBar, SearchInput,SearchWrapper, SearchLogo} from '../../styled/Header'
 
 
 export default class Searchbar extends Component {
     render() {
         return (
-            <nav
-                className="navbar navbar-expand-md navbar-light bg-white sticky-top justify-content-md-between"
+            <Header
+                className="navbar navbar-expand-md navbar-light d-md-flex justify-content-md-between"
             >
                 <IconContext.Provider value={{size:'32px'}}>
-                    <a
+                    <Logo
                         href="/index.html"
                         className="navbar-brand"
                     >
                         <FaInstagram /> | Instagram
-                    </a>
-                    <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    </Logo>
+                    <button className="navbar-toggler " type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                         <span className="navbar-toggler-icon"></span>
                     </button>
-                    <section className="collapse navbar-collapse d-md-flex justify-content-md-between" id="navbarSupportedContent">
-                        <section className="search">
-                            <FaSearch/>
-                            <input type="text" placeholder="Search" onChange={this.props.searchEdit} value={this.props.searchValue}/>
-                        </section>
+                    <SearchBar className="collapse navbar-collapse" id="navbarSupportedContent">
+                        <SearchWrapper className="offset-md-3">
+                            <SearchLogo />
+                            <SearchInput type="text" placeholder="Search" onChange={this.props.searchEdit} value={this.props.searchValue}/>
+                        </SearchWrapper>

```

Comment:
Interesting to see you add the logo in the search bar by wrapping it as such. Nice work.